index.bs

```diff
@@ -76,773 +93,795 @@ All <{iframe}> use cases are still valid and should continue to use <{iframe}>,
 including IWAs where possible.
 
 <!-- ====================================================================== -->
-# Usage Overview # {#usage-overview}
-<!-- ====================================================================== -->
-
-Lorem ipsum. Insert basic info and example here.
-
-<!-- ====================================================================== -->
-# Motivating Applications # {#motivating-applications}
+<h2 id=based-on-fencedframe-spec>The Fenced Frame specification</h2>
 <!-- ====================================================================== -->
 
-*This section is non-normative.*
-
-<!-- ====================================================================== -->
-## Latency-sensitive applications in virtualized sessions ## {#browser-content-redirection}
-<!-- ====================================================================== -->
+For convenience, the Controlled Frame specification assumes that the Fenced
+Frame specification is in place. There are concepts introduced in the Fenced
+Frame specification, such as top-level traversibles, that are broadly useful
```

Comment:
I believe top-level traversibles are already in the HTML spec. My reading is that fenced frames are introducing nested top-level navigables.

Reply:
Done.